index.js

```diff
@@ -74,6 +74,12 @@ var WebpackBuildNotifierPlugin = function(cfg) {
      * Set to false to play no sound for failure notifications. Takes precedence over the *sound* configuration option.
      */
     this.failureSound = cfg.hasOwnProperty('failureSound') ? cfg.failureSound : this.sound;
+    /**
+     * @cfg {String} [compilationSound='Submarine']
+     * The sound to play for compilation notifications. Defaults to the value of the *sound* configuration option.
+     * Set to false to play no sound for compilation notifications. Takes precedence over the *sound* configuration option.
+     */
+    this.compilationSound = cfg.hasOwnProperty('compilationSound') ? cfg.compilationSound : this.sound;
```

Comment:
I'm thinking it may be a better idea for this _not_ to have a default value. This would mean users would need to explicitly set this option to get a sound with the compilation notification, but this way we aren't spamming existing users with submarine sounds on each compile. :) Thoughts?

Reply:
If you think `false` is a better default, we can do that - but since all the other sounds use the general sound option as default, I picked that. Another reason in favor of this: Changing the `sound` option should affect **all** sounds, not just all *except* compilation notification. 
Since compilation notifications are off by default, this change will not annoy those who do not have them turned on, but for those who do it's likely a welcome change that `sound: false` now also affects compile start, and that it can be changed individually.

Reply:
Hmm, good points. My reservation with this approach is that those who have compilation notifications enabled and typically run their builds with watch enabled will now get sounds for each compilation (which could be quite frequent!), and I know this would annoy me personally.

That said, it's easy enough to disable and I think consistency wins out here. Will merge as is, thanks again!

Reply:
I just tried setting `sound: false` in the plugin config on my Windows 10 Pro machine, and it disabled all sounds as expected. Not working for you?

Reply:
It wasn't on 0.1.28, that's why I made this fix.
"My reservation with this approach is that those who have compilation notifications enabled and typically run their builds with watch enabled will now get sounds for each compilation (which could be quite frequent!), and I know this would annoy me personally." This implies that the compilation notifications had no sound on your machine, but on my Win10 they did with no way to disable it, as `sound` didn't affect that.

Reply:
Ah, gotcha! In other words, it should be working fine in 0.1.29. I'm a little slow today.

Thanks again for taking this up.